### DISCUSSION TITLE
Fix inconsistent tooltip capitalization (again)

### DISCUSSION THREAD

**Discussion by revelsix** [ppy/osu#21764] [Ideas]
This was already discussed and attempted in #7515 and #7534, but it looks like whichever person tried to fix it missed a ton of tooltips 🙄.

This fix requires changes from 3 different places.

- [ ] Non-localized strings (small, easy fix) #21765 
- [ ] Localized game strings (requires update to localization files in-game) same PR as above.
- [ ] Localized web strings (requires update to lang files in ppy/osu-web) ppy/osu-web#9669

It also raises a couple questions.

- How should longer tooltips be handled?
Compare `attempts to bypass any operation system mouse acceleration. on windows, this is equivalent to what used to be known as &quot;raw input&quot;.` with `Attempts to bypass any operation system mouse acceleration. On windows, this is equivalent to what used to be known as &quot;Raw Input&quot;.` Ignoring "operation system" and a lowercase w for Windows in the capitalized string, which one is preferred?

- How will consistency between languages be handled?
This was the main objection brought up when discussing this on the discord, and according to [Susko3](https://discord.com/channels/188630481301012481/188630652340404224/1055441762149216379), the two main solutions are invalidating existing translations or silently updating the english string where translators are just given a notification with existing translations staying unchanged

**Comment by bdach** [comment]
No changes should be applied to osu-resources directly. Please read https://github.com/ppy/osu/wiki/Localising-the-client.
